package-lock.json

```diff
@@ -1307,6 +1308,155 @@
       "integrity": "sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==",
       "dev": true
     },
+    "node_modules/@cloudscape-design/browser-test-tools": {
```

Comment:
the @cloudscape-design dependencies shouldn't be there

Reply:
Should this be removed by this script?
https://github.com/cloudscape-design/board-components/blob/main/package.json#L125